<launch>

	<node pkg="waypoint_follower" type="mpc_waypoints_converter" name="mpc_waypoints_converter" output="screen"/>

	<node pkg="waypoint_follower" type="mpc_follower" name="mpc_follower" output="screen">
		<param name="ctrl_cmd_interface" value="all"/>
		<param name="in_waypoints_name" value="/mpc_waypoints"/>
		<param name="in_vehicle_status_name" value="/vehicle_status"/>
		<param name="in_selfpose_name" value="/current_pose"/>
		<param name="out_twist_name" value="/twist_cmd"/>
		<param name="out_vehicle_cmd_name" value="/ctrl_cmd"/>
	</node>

	<node pkg="temporal_monitor" type="temporal_monitor_node" name="temporal_monitor"/>
	<node pkg="simple_obstacle_avoidance" type="simple_obstacle_avoidance_node" name="simple_obstacle_avoidance"/>

</launch>
